### PR TITLE
Solves #34: Show not allowed civs

### PR DESCRIPTION
**Pull request by Des-**
Solves #34 .

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -113,6 +115,7 @@ final class config
         self::draw_player_num_civs_3vs3 => 4,
         self::draw_player_num_civs_4vs4 => 3,
         self::draw_factor => 0.4,
+        self::free_pick_civ_id => 32,
```
Comment (mcepok) [inline config/config.php]: Please rather use 0 as default value.

File: adm/style/acp_nczone_general.html [hunk 2]
```diff
@@ -92,6 +92,15 @@
 			<textarea style="width:200px;height:100px;" name="nczone_info_posts">{{ nczone_info_posts }}</textarea>
 		</dd>
 	</dl>
+	<dl>
+		<dt>
+			<label for="nczone_free_pick_civ_id">{{ lang('NCZONE_FREE_PICK_CIV_ID') }}{{ lang('COLON') }}</label><br />
+			<span>{{ lang('NCZONE_FREE_PICK_CIV_ID_DESCR') }}</span>
+		</dt>
+		<dd>
+			<input type="number" step="1" min="1" name="nczone_free_pick_civ_id" value="{{ nczone_free_pick_civ_id }}" />
```
Comment (mcepok) [inline adm/style/acp_nczone_general.html]: Allow 0 to disable this functionality

File: zone/matches.php [hunk 3]
```diff
@@ -727,6 +727,67 @@ public function get_team_civs(int $team1_id, int $team2_id, int $map_id): array
         return $teams;
     }
 
+    private static function get_free_pick_civ_id()
```
Comment (mcepok) [inline zone/matches.php]: This can be inlined

File: zone/matches.php [hunk 4]
```diff
@@ -727,6 +727,67 @@ public function get_team_civs(int $team1_id, int $team2_id, int $map_id): array
...
+    {
+        return config::get(config::free_pick_civ_id);
+    }
+
+    public function get_banned_civs(int $match_id, int $map_id): array
+    {
+        $freePickCivId = strval($this->get_free_pick_civ_id());
```
Comment (mcepok) [inline zone/matches.php]: Please don't use camel case for PHP variables

File: adm/style/acp_nczone_general.html (hunk 2 above)
Comment (Des-) [reply adm/style/acp_nczone_general.html]: done

File: zone/matches.php (hunk 3 above)
Comment (Des-) [reply zone/matches.php]: done

File: zone/matches.php (hunk 4 above)
Comment (Des-) [reply zone/matches.php]: bäääääääh

**Pull request merged**